checkbox-provider-ce-oem/tests/test_thermal_sensor_test.py

```diff
@@ -0,0 +1,77 @@
+import unittest
```

Comment:
Don't we need a `#!/usr/bin/env pyhon3`?

Reply:
we don't need the shebang in the unit test file.